test/expected/build/library.j/dest/resources/library/j/dependency-es6-2.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * ${copyright}
+ */
+
+/**
+ * Covers:
+ * - ArrowFunctionExpression
+ */
+window.someRandomModule ||
+	sap.ui.define(["/.a"], (a) =>
+		a.extend(`aaa`, {
```

Comment:
Should we also add actual JSDoc comments for all those cases so that we can verify whether the api.json contains the expected content? With this change the api.json stays the same.

Reply:
Now should be ok